### ISSUE TITLE
BUG: Say something here

### ISSUE THREAD

**Issue by thiagola92** [beekeeper-studio/beekeeper-studio#327]
**Describe the bug**
Quoting SQL keywords and wrong quoting columns and table names.

```sql
CREATE TABLE "ASDF.A.B" ("a.b" text)
CREATE TABLE asdf ("a.b" text)
CREATE TABLE "asdf.a" (asdf text)
CREATE TABLE "texto com espaco" (abc text)

-- Try autocomplete SELECT
-- expectation: SELECT
-- outcome: "SELECT"
SEL

-- Try autocomplete table name with dot or space or capital letter
-- expectation: "ASDF.A.B"
-- outcome: "ASDF"."A"
SELECT * FROM A

-- expectation: "texto com espaco"
-- outcome: texto com espaco
SELECT * FROM tex

-- Try autocomplete column name with dot
-- expectation: asdf."a.b"
-- outcome: asdf.a
SELECT asdf.

-- Try autocomplete column name after quoted table name
-- expectation: "asdf.a".asdf
-- outcome: no result
SELECT "asdf.a". FROM "asdf.a"
```

**To Reproduce**
1. Connect to **postgres** database
2. Run the first 4 lines from the code above
3. Try autocomplete the code

**Expected behavior**
Expected not to quote keywords  
Expected quote only table/column name that have dot, capital letter or space
Expected to find the column name after quoted table name

**Screenshots / Gifs**
https://www.youtube.com/watch?v=6_1GUrd7zCA

**Version Information (please complete the following information):**
 - OS: Ubuntu 20.04
 - App Version 1.7.5

**Comment by thiagola92** [comment]
On postgres when you want your table/column name to have dot/capital letter/space you have to quote it.  
`CREATE TABLE "EXAMPLE"(name text)`  
`CREATE TABLE "example.two"(name text)`  
`CREATE TABLE "example three"(name text)`  

Right now beekeeper try to decide if need to quote or not the text received from CodeMirror.  
https://github.com/beekeeper-studio/beekeeper-studio/blob/c13ee7274d61741a451a127559f3e8c75e37c7db/src/components/TabQueryEditor.vue#L526-L535  

The problem start when CodeMirror shows table/columns names that should have quote, without quote on the autocomplete  
https://github.com/beekeeper-studio/beekeeper-studio/blob/c13ee7274d61741a451a127559f3e8c75e37c7db/src/components/TabQueryEditor.vue#L222-L234  

If CodeMirror shows **foo.bar**, it could be two options:  
Exist a table/column named that way: **"foo.bar"**  
You want to get the column "bar" from table "foo": **foo.bar**

**Comment by rathboma** [comment]
Yes there are unfortunately a lot of limitations with the autocomplete right now. 

Thanks for digging into the code a little bit.
